radio/src/targets/9x/lcd_driver.cpp

```diff
@@ -145,6 +192,41 @@ uint8_t lcdRefresh_ST7920(uint8_t full)
 void lcdRefresh()
 {
 #endif
+#if defined(LCD_KS108)
+static uint8_t change = 0; // toggle left or right lcd writing

```

Comment:
My mind was to have the fastest lcd function. They are two controllers in a KS108 256X64 and the driver toggle left and right (2.5mS by lcdrefresh call) .... But all popup needs two call for drawing correctly. The result of my mind with this LCD : More time for the mixer and less power consumption.